CODEOWNERS

```diff
@@ -261,15 +261,15 @@ geode-core/**/org/apache/geode/internal/datasource/**             @agingade @Ben
 #-----------------------------------------------------------------
 # Resource manager
 #-----------------------------------------------------------------
-#geode-core/**/org/apache/geode/cache/control/**
-#geode-core/**/org/apache/geode/internal/cache/control/**
```

Comment:
To avoid having a single owner for these, you can add me if you like. Or perhaps @DonalEvans would be a good candidate since there is some rebalancing code in here?

Reply:
I have added @DonalEvans as a reviewer and added him to the Resource Manager lines.